src/app/components/Document/index.jsx

```diff
@@ -27,12 +26,7 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
   return (
     <html lang="en-GB" {...htmlAttrs}>
       <head>
-        <meta httpEquiv="X-UA-Compatible" content="IE=edge" />
-        <meta charSet="utf-8" />
-        <meta name="robots" content="noindex,nofollow" />
-        <meta name="theme-color" content={C_POSTBOX} />
         {meta}
-        <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
         <link rel="manifest" href="/manifest.json" />
```

Comment:
Discussed this with Sareh in person -- the logic for keeping this link to `manifest.json` in the Document, & not moving it to an optional prop in Metadata, is that we think we'll want to be able to disable the SPA aspects of Simorgh in a single step, & not require all users of Psammead to worry about it.